cunumeric/module.py

```diff
@@ -2234,7 +2237,7 @@ def tile(A: ndarray, reps: Union[int, ndarray]) -> ndarray:
     # Prepend ones until the dimensions match
     while len(computed_reps) < out_dims:
         computed_reps = (1,) + computed_reps
-    out_shape: NdShape = ()
+    out_shape: tuple[Any, ...] = ()
```

Comment:
Seems this can be nested? `SupportsIndex` might work for this but it seems uncommon

Reply:
This should always be an array shape. I cleaned up the code to make this clear.